### ISSUE TITLE
Installation failed: Failed to download vc_redist.x64.exe

### ISSUE THREAD

**Issue by rilut** [picotorrent/picotorrent#302]
```
[2FF8:2CBC][2016-06-06T18:30:40]w343: Prompt for source of package: VC_redist.x64.exe, payload: VC_redist.x64.exe, path: C:\Users\RILUT\Downloads\VC_redist.x64.exe
[2FF8:2CBC][2016-06-06T18:30:40]i338: Acquiring package: VC_redist.x64.exe, payload: VC_redist.x64.exe, download from: https://download.microsoft.com/download/C/E/5/CE514EAE-78A8-4381-86E8-29108D78DBD4/VC_redist.x64.exe
[2FF8:2CBC][2016-06-06T18:30:41]e000: Error 0x80070002: Failed attempt to download URL: 'https://download.microsoft.com/download/C/E/5/CE514EAE-78A8-4381-86E8-29108D78DBD4/VC_redist.x64.exe' to: 'C:\Users\RILUT\AppData\Local\Temp\{88818ba2-ab1f-49a9-82d2-1af3f71cbb40}\VC_redist.x64.exe'
[2FF8:2CBC][2016-06-06T18:30:41]e000: Error 0x80070002: Failed to acquire payload from: 'https://download.microsoft.com/download/C/E/5/CE514EAE-78A8-4381-86E8-29108D78DBD4/VC_redist.x64.exe' to working path: 'C:\Users\RILUT\AppData\Local\Temp\{88818ba2-ab1f-49a9-82d2-1af3f71cbb40}\VC_redist.x64.exe'
[2FF8:2CBC][2016-06-06T18:30:41]e313: Failed to acquire payload: VC_redist.x64.exe to working path: C:\Users\RILUT\AppData\Local\Temp\{88818ba2-ab1f-49a9-82d2-1af3f71cbb40}\VC_redist.x64.exe, error: 0x80070002.
[2FF8:2730][2016-06-06T18:30:41]e000: Error 0x80070002: Failed while caching, aborting execution.
[24B8:0D28][2016-06-06T18:30:41]i330: Removed bundle dependency provider: {88818ba2-ab1f-49a9-82d2-1af3f71cbb40}
[24B8:0D28][2016-06-06T18:30:41]i352: Removing cached bundle: {88818ba2-ab1f-49a9-82d2-1af3f71cbb40}, from path: C:\ProgramData\Package Cache\{88818ba2-ab1f-49a9-82d2-1af3f71cbb40}\
[2FF8:2730][2016-06-06T18:30:41]i399: Apply complete, result: 0x80070002, restart: None, ba requested restart:  No
```

I think https://download.microsoft.com/download/C/E/5/CE514EAE-78A8-4381-86E8-29108D78DBD4/VC_redist.x64.exe is dead

**Comment by rilut** [comment]
https://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe is okay

By the way, how do you extract information for `PicoTorrentBundle.wxs`?

**Comment by vktr** [comment]
Yeah, I've noticed that too. Don't know why MS removed the old installer... I've fixed it for the next version.

The information is extracted manually.

**Comment by vktr** [comment]
Closing this due to it being fixed in 795565f3cf3dd3e2ab58f1976331bf22a57864d7 and will be released in vNext.
